tests/conftest.py

```diff
@@ -0,0 +1,233 @@
+import contextlib
+import dataclasses
+import os
+import pathlib
+import re
+import shlex
+import shutil
+import subprocess
+import sys
+import tempfile
+import textwrap
+
+import pytest
+import pexpect
+
+
+DEFAULT_TIMEOUT = 3  # in seconds
+
+@pytest.fixture(scope="session")
```

Comment:
```suggestion
@pytest.fixture(scope="function")
```

home has some state that you likely want to be cleaned up between different tests.
Ive seen that especially for remote env tests